src/theme/components/textInput.ts

```diff
@@ -38,14 +39,26 @@ const input = {
 
 const TextInput = {
   parts: ["input", "textarea"],
-  baseStyle: {
+  baseStyle: ({ showLabel }) => ({
+    position: "relative",
     input,
     textarea: {
       ...input,
       lineheight: "1.5",
       minHeight: "xxl",
     },
-  },
+    clearButton: {
```

Comment:
The default padding is not great, so add the following:
```
px: "xs"
```

Reply:
This is odd but updating this does not affect the padding. Check it out. I think it's because of the flex display inside of the button.